src/validation.cpp

```diff
@@ -3523,7 +3523,7 @@ bool static LoadBlockIndexDB(const CChainParams& chainparams)
     // Check whether we need to continue reindexing
     bool fReindexing = false;
     pblocktree->ReadReindexing(fReindexing);
-    fReindex |= fReindexing;
```

Comment:
What's wrong with |= ?

Reply:
See my reply to @TheBlueMatt's comment above, it's not implemented for atomic bool :)